# Navigate.srv
string target_location
---
bool reach
string message